clock_exercise/alarmitemmodel.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <QAbstractListModel>
+#include "alarmitemdata.h"
+#include <QDate>
+
+class AlarmItemModel : public QAbstractListModel
+{
+    Q_OBJECT
+
+    Q_PROPERTY(int numberEverydayAlarms MEMBER m_numberEverydayAlarms NOTIFY numberEverydayAlarmsChanged WRITE setNumberEverydayAlarms)
+    Q_PROPERTY(int numberSelectedAlarms MEMBER m_numberSelectedAlarms NOTIFY numberSelectedAlarmsChanged WRITE setNumberSelectedAlarms)
```

Comment:
Non sono convintissimo di avere una property del genere lato C++, considerando che l'unico motivo per cui ce l'abbiamo è quello di cambiare una `visibility` in QML per il tasto di delete...